src/Core/src/Platform/iOS/MauiDatePicker.cs

```diff
@@ -1,13 +1,69 @@
-﻿using ObjCRuntime;
+﻿using System;
+using Foundation;
 using UIKit;
 
 namespace Microsoft.Maui.Platform
 {
 	public class MauiDatePicker : NoCaretField
 	{
+#if !MACCATALYST
```

Comment:
Is this too much negative? 😜 Would `#if iOS` be more readable? Just saying that someone could miss the "!" or, is this planned if we support tvOS or watchOS? 
_Not a mandatory change_

Reply:
https://github.com/dotnet/maui/pull/8476#discussion_r918255236